### PR TITLE
OSDOCS-7900: Documented the 4.11.50 z-stream release notes

### PR DESCRIPTION
**Pull request by dfitzmau**
[OSDOCS-7900](https://issues.redhat.com/browse/OSDOCS-7900)

Version(s):
4.11

Link to docs preview:
[4.11.50](https://65225--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-11-release-notes#ocp-4-11-50)

QE review:
- [X] QE not required

Additional information:

### CODE REVIEW

**Review by cbippley — COMMENTED** [review]
LGTM! Please wait for @opayne1 to review before requesting merge review since I'm still in training, thanks!

File: release_notes/ocp-4-11-release-notes.adoc
Comment (opayne1) [inline release_notes/ocp-4-11-release-notes.adoc:3767]:
```suggestion
* Previously, the cloud credentials used in the Manila CSI Driver Operator were cached, resulting in authentication issues if these credentials were rotated. With this update, this issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-18782[*OCPBUGS-18782*])
```

**Review by opayne1 — COMMENTED** [review]
Just one tiny nit but otherwise lgtm

File: release_notes/ocp-4-11-release-notes.adoc
Comment (JoeAldinger) [inline release_notes/ocp-4-11-release-notes.adoc:3754]:
```suggestion
Issued: 2023-10-04
```

File: release_notes/ocp-4-11-release-notes.adoc
Comment (JoeAldinger) [inline release_notes/ocp-4-11-release-notes.adoc:3764]: This needs to be included like in 4.13 and 4.12 https://issues.redhat.com/browse/OCPBUGS-18545 as a feature write up that is a backport. [This](https://docs.openshift.com/container-platform/4.13/release_notes/ocp-4-13-release-notes.html#ocp-4-13-12-custom-rhcos-image-for-gcp) content needs to be added to this RN.

**Review by JoeAldinger — CHANGES_REQUESTED** [review]
@dfitzmau please make sure to include the RHCOS content so we don't have to do CM.

**Review by JoeAldinger — CHANGES_REQUESTED** [review]
Small organizational move

**Review by JoeAldinger — APPROVED** [review]

**Pull request merged**
